x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/patch_rules_route.ts

```diff
@@ -69,6 +69,8 @@ export const patchRulesRoute = (router: SecuritySolutionPluginRouter, ml: SetupP
           throwAuthzError(await mlAuthz.validateRuleType(existingRule?.params.type));
         }
 
+        checkDefaultRuleExceptionListReferences({ exceptionLists: params.exceptions_list });
```

Comment:
Same question about catching the error and returning with `siemResponse.error()`